cli/command/stack/kubernetes/list.go

```diff
@@ -19,11 +20,18 @@ import (
 // GetStacks lists the kubernetes stacks
 func GetStacks(kubeCli *KubeCli, opts options.List) ([]*formatter.Stack, error) {
 	if opts.AllNamespaces || len(opts.Namespaces) == 0 {
+		if isAllNamespacesDisabled(kubeCli.ConfigFile().Kubernetes) {
+			opts.AllNamespaces = true
+		}
 		return getStacksWithAllNamespaces(kubeCli, opts)
 	}
 	return getStacksWithNamespaces(kubeCli, opts, removeDuplicates(opts.Namespaces))
 }
 
+func isAllNamespacesDisabled(kubeCliConfig *configfile.KubernetesConfig) bool {
+	return kubeCliConfig == nil || kubeCliConfig != nil && kubeCliConfig.AllNamespaces != "disabled"
```

Comment:
Oh, one thing: do we want a strict check to only accept "disabled", "enabled" or `nil` ?

Reply:
yes, @silvin-lubecki will follow up :angel:

Reply:
Sure I will!